src/test/java/com/rocketden/main/mapper/RoomMapperTests.java

```diff
@@ -56,6 +80,12 @@ public void entityToDto() {
         assertEquals(room.getDuration(), response.getDuration());
         assertEquals(room.getNumProblems(), response.getNumProblems());
 
+        assertEquals(room.getProblems().size(), response.getProblems().size());
+        assertEquals(room.getProblems().get(0).getProblemId(), response.getProblems().get(0).getProblemId());
+        assertEquals(room.getProblems().get(0).getName(), response.getProblems().get(0).getName());
+        assertEquals(room.getProblems().get(1).getDifficulty(), response.getProblems().get(1).getDifficulty());
```

Comment:
It might make more sense to instead use `problem1.getProblemId()`, `problem1.getName()`, and `problem2.getDifficulty()`. Also, how did you decide to compare those specific attributes, since you're not testing all of them for both?

Reply:
Done! And yeah it's essentially randomly tested for breadth - it might be a bit repetitive to test all fields for both problems, so I tested various fields across both